ikenna-portfolio/portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```

Comment:
Is this the same content as target/portfolio-1/projects.html?

Reply:
I believe so, I can't see anything that shows they're different.